manim/scene/vector_space_scene.py

```diff
@@ -698,12 +663,7 @@ def add_transformable_mobject(self, *mobjects):
         self.add_special_mobjects(self.transformable_mobjects, *mobjects)
 
     def add_moving_mobject(self, mobject, target_mobject=None):
-        """
-        Adds the mobject to the special list
-        self.moving_mobject, and adds a property
-        to the mobject called mobject.target, which
-        keeps track of what the mobject will move to
-        or become etc.
+        """Add the mobject to the special list self.moving_mobject, and adds a property to the mobject called mobject.target, which keeps track of what the mobject will move to or become etc.
```

Comment:
Also needs a summary that I can't think of right now.

Reply:
```suggestion
        """Add the property mobject.target to target_mobject. Add mobject to a list for tracking interpolation to target_mobject.
```